static/page-templates/page-templates.json

```diff
@@ -2,84 +2,94 @@
-			"defaults": "maywood",
-			"vertical-whitelist": []
-		},
-		{
-			"title": "rivington",
-			"defaults": "rivington",
-			"vertical-whitelist": []
+			"title": "Vesta",
+			"slug": "vesta",
+			"template": "mayland2",
+			"theme": "mayland",
+			"src": "https://public-api.wordpress.com/rest/v1/template/demo/mayland/mayland2/",
+			"fonts": [ "Cabin", "Raleway" ],
+			"categories": [ "featured", "portfolio" ]
+		},
+		{
+			"title": "WIP Event Template",
+			"slug": "rivington",
+			"template": "rivington",
+			"theme": "rivington",
+			"src": "https://public-api.wordpress.com/rest/v1/template/demo/rivington/rivington/",
+			"fonts": [ "Arvo", "Montserrat" ],
+			"categories": [ "featured", "event" ]
+		},
+		{
+			"title": "Reynolds",
+			"slug": "reynolds",
+			"template": "rockfield2",
+			"theme": "rockfield",
+			"src": "https://public-api.wordpress.com/rest/v1/template/demo/rockfield/rockfield2/",
+			"fonts": [ "Playfair Display", "Fira Sans" ],
+			"categories": [ "featured", "portfolio" ]
+		},
+		{
+			"title": "Easley",
+			"slug": "easley",
+			"template": "maywood",
+			"theme": "maywood",
+			"src": "https://public-api.wordpress.com/rest/v1/template/demo/maywood/maywood/",
+			"fonts": [ "Space Mono", "Roboto" ],
+			"categories": [ "featured", "portfolio" ]
+		},
+		{
+			"title": "Camden",
+			"slug": "Camden",
+			"template": "maywood2",
+			"theme": "maywood",
+			"src": "https://public-api.wordpress.com/rest/v1/template/demo/maywood/maywood2/",
+			"fonts": [ "Space Mono", "Roboto" ],
+			"categories": [ "featured", "portfolio" ]
+		},
+		{
+			"title": "Bowen",
+			"slug": "bowen",
+			"template": "bowen",
+			"theme": "coutoire",
+			"src": "https://public-api.wordpress.com/rest/v1/template/demo/coutoire/bowen/",
+			"fonts": [ "Playfair Display", "Fira Sans" ],
+			"categories": [ "featured", "blog" ]
+		},
+		{
+			"title": "Edison",
+			"slug": "edison",
+			"template": "stratford2",
+			"theme": "stratford",
+			"src": "https://public-api.wordpress.com/rest/v1/template/demo/stratford/stratford2/",
+			"fonts": [ "Chivo", "Open Sans" ],
+			"categories": [ "featured", "blog" ]
+		},
+		{
+			"title": "Cassel",
+			"slug": "cassel",
+			"template": "mayland",
+			"theme": "mayland",
+			"src": "https://public-api.wordpress.com/rest/v1/template/demo/mayland/mayland/",
+			"fonts": [ "Playfair Display", "Fira Sans" ],
+			"categories": [ "featured", "blog" ]
+		},
+		{
+			"title": "Overton",
+			"slug": "overton",
+			"template": "alves",
+			"theme": "alves",
+			"src": "https://public-api.wordpress.com/rest/v1/template/demo/alves/alves/",
+			"fonts": [ "Cabin", "Raleway" ],
+			"categories": [ "featured", "business" ]
+		},
+		{
+			"title": "Doyle",
+			"slug": "doyle",
+			"template": "alves2",
+			"theme": "alves",
+			"src": "https://public-api.wordpress.com/rest/v1/template/demo/alves/alves2/",
+			"fonts": [ "Playfair Display", "Fira Sans" ],
+			"categories": [ "featured", "business" ]
 		}
 	],
 	"themes": [
```

Comment:
While we're here, is it safe to remove the `themes` array? In D41177-code, I changed the template demo endpoint to infer a given theme's demo site through `wpcom_get_theme_demo_site()`. If that was the only consumer of the `themes` array, it should be safe to remove now. (Or was there another?)

Reply:
Oh, sorry, I haven't seen that. Yeah, that array was used only to match with the demo site, so I guess it's safe to remove it now!